scripts/mkcloud

```diff
@@ -324,6 +324,13 @@ function cleanup()
     if [ -d $vdisk_dir ]; then
         find -L $vdisk_dir -name "$cloud.*" -type b | \
             xargs -r lvremove --force
+        if [ $? != 0 ]; then
+            echo >&2
+            echo "Failed to free up previous volumes:" >&2
+            echo >&2
+            find -L $vdisk_dir -name "$cloud.*" -type b
+            exit 42
```

Comment:
Because `"some error message"` needs to be multi-line and include output from the command, and shell's string handling is bad enough that there's no way to do this cleanly.